.env

```diff
@@ -1 +1,2 @@
- MONGO_URI = mongodb://northUser:bGJucFTFliN8qX2Z@ac-gkbjsml-shard-00-00.oxafelt.mongodb.net:27017,ac-gkbjsml-shard-00-01.oxafelt.mongodb.net:27017,ac-gkbjsml-shard-00-02.oxafelt.mongodb.net:27017/?ssl=true&replicaSet=atlas-yrtq9t-shard-0&authSource=admin&retryWrites=true&w=majority
+MONGO_URI = mongodb://northUser:bGJucFTFliN8qX2Z@ac-gkbjsml-shard-00-00.oxafelt.mongodb.net:27017,ac-gkbjsml-shard-00-01.oxafelt.mongodb.net:27017,ac-gkbjsml-shard-00-02.oxafelt.mongodb.net:27017/?ssl=true&replicaSet=atlas-yrtq9t-shard-0&authSource=admin&retryWrites=true&w=majority
+JWTKEY="32CHARSTRING32CHARSTRING32CHARSTRING"
```

Comment:
This information should never be included in git, because it means that anyone who has access to the git repo (which in this case is public) can steal the database credentials (and all of the data therein).

I recommend using the following pattern for environment variable files in Next.js (but it's really up to the team, and many solutions are fine, as long as secrets like these don't make it into git):

* Use `.env.development` for development secrets, include the secrets in the file (which is fine because they are only for the development system) and **include the file in git**
* Use `.env.production` for **non-secret** production variables, such as hostnames, configuration etc, and **include the file in git**.
* Use `.env.local` to override secrets, especially in production and **do not include in git**.